vsintegration/tests/UnitTests/InlineParameterNameHintTests.fs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace VisualFSharp.UnitTests.Editor.Hints
+
+open NUnit.Framework
+open HintTestFramework
+
+module InlineParameterNameHintTests =
```

Comment:
Please add a negative test for demonstrating that parameters with explicitely specified types (p:int) do not get a parameter hint.
(or at least I believe they should not get it).